elasticms-admin/.env.dist

```diff
@@ -61,7 +63,7 @@ EMSCO_INSTANCE_ID=elasticms_
 EMSCO_NAME='<i class="ems-icon-ball"></i> elastic<b>ms</b>'
 EMSCO_SHORTNAME='<i class="ems-icon-ball"></i><span class="sr-only">elasticms</span>'
 EMSCO_SINGLE_TYPE_INDEX=false
-EMSCO_THEME_COLOR='blue'
+EMSCO_THEME_COLOR='yellow'
```

Comment:
use purple. blue is for prod, yellow integration. green acc and red for test. Purple is dedicated to dev